src/goto-programs/goto_function.h

```diff
@@ -17,13 +20,19 @@ Date: May 2018
 
 #include "goto_program.h"
 
+/// A goto function, consisting of function type, function body, and parameter
+/// identifiers.
```

Comment:
A link to `goto_programt` should be provided to enable readers to figure out more details.